openstack/compute/v2/extensions/simpletenantusage/doc.go

```diff
@@ -0,0 +1,28 @@
+/*
+Package simpletenantusage provides information and interaction with the
+SimpleTenantUsage extension for the OpenStack Compute service.
+
+Example to Retrieve Usage for a Single Tenant:
+
+	start := time.Date(2017, 01, 21, 10, 4, 20, 0, time.UTC)
+	end := time.Date(2017, 01, 21, 10, 4, 20, 0, time.UTC)
```

Comment:
minor nit: spacing

Reply:
I don't know what you mean here
https://play.golang.org/p/CjyvDX94Cv

Reply:
I apologize. What I meant was that these two lines are indented further in than they should be:

https://github.com/hypermassiveblackhole/gophercloud/blob/4e16d78279dc6aa8f42a662a224a3de5bcfb634f/openstack/compute/v2/extensions/simpletenantusage/doc.go